docker/docker-compose.yml

```diff
@@ -112,4 +128,5 @@ volumes:
             o: bind
             # This directory specified where InvenTree data are stored "outside" the docker containers
             # Change this path to a local system path where you want InvenTree data stored
-            device: /path/to/data
\ No newline at end of file
+            device: /Users/markdedeuge/data
```

Comment:
probably don't want to commit this one...

Reply:
Good catch.

Reply:
@SchrodingersGat should I add the docker-compose parameterisation into this PR?

It would replace these variables, ie `/path/to/xxx` would be replaced with an `${INVENTREE_SOME_VAR:?inventree_some_var}` and the user would then only look in the default_env file to see what needs updating.

Reply:
I'd like to get this one merged first, it's quickly blowing out into a huge change!

Reply:
But moving these things to the .env file is a good idea.